Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -333,6 +333,28 @@ static String mutationObserverNotificationString()
 
     if (m_pdfTestCallback)
         std::exchange(m_pdfTestCallback, nullptr)->handleEvent();
+
+    sizeToFitContents();
+}
+
+bool UnifiedPDFPlugin::pluginFitsContent() const
+{
+#if PLATFORM(IOS_FAMILY)
+    return true;
+#else
+    return false;
+#endif
+}
+
+void UnifiedPDFPlugin::sizeToFitContents()
+{
+    if (isLocked())
+        return;
+    auto size = contentsSize();
+
+    auto& pluginElement = m_view->pluginElement();
+    pluginElement.setInlineStyleProperty(CSSPropertyWidth, size.width(), CSSUnitType::CSS_PX);
+    pluginElement.setInlineStyleProperty(CSSPropertyHeight, size.height(), CSSUnitType::CSS_PX);
```

Comment:
```suggestion
    Ref pluginElement = m_view->pluginElement();
    pluginElement->setInlineStyleProperty(CSSPropertyWidth, size.width(), CSSUnitType::CSS_PX);
    pluginElement->setInlineStyleProperty(CSSPropertyHeight, size.height(), CSSUnitType::CSS_PX);
```

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/e6abaffdcea753bb1711d92bc067f42526af4c96